src/kibana/components/config/defaults.js

```diff
@@ -43,6 +43,21 @@ define(function (require) {
       value: 6,
       description: 'The maximum geoHash size allowed in a tile map',
     },
+    'visualization:tileMap:wms': {
+      value: false,
+      description: 'Set for WMS mapping provider'
```

Comment:
Can you provide a more complete description of this? Maybe a link to a reference?